### ISSUE TITLE
I hope, this repository isnt dead?

### ISSUE THREAD

**Issue by Camper12345** [Darmo117/SpecialBlockMovements#1]
Is this mod works on forge?

**Comment by DamiaV** [comment]
Yes, it is developed with Forge. There haven't been any commits lately because I was on other projects but it's not abandonned. For now, there's not much in the mod but I plan to add more over time.
